content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -178,6 +192,315 @@ Error from server (NotFound): Unable to list {"stable.example.com" "v1" "crontab
 
 If you later recreate the same CustomResourceDefinition, it will start out empty.
 
+## Specifying a structural schema
+
+{{< feature-state state="beta" for_kubernetes_version="1.15" >}}
+
+CustomResources traditionally store arbitrary JSON (next to `apiVersion`, `kind` and `metadata`, which is validated by the API server implicitly). With [OpenAPI v3.0 validation](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation) a schema can be specified, which is validated during creation and updates, compare below for details and limits of such a schema.
```

Comment:
Should we say JSON here? It's stored that way, but from the user perspective, it can be JSON or YAML..

Reply:
s/traditionally/"early versions of CustomResources"?

Reply:
everything up to 1.15 and everything not setting `spec.preserveUnknownFields: false`

Reply:
Hm, that might be too wordy. "traditionally" might be okay..